src/content/advanced/upgrades/aws-19-release/index.md

```diff
@@ -59,6 +61,30 @@ For further information, please checkout the [documentation](https://kubernetes-
 While switching to Cilium we are forced to change the CIDR used to assign IPs to Pods (192.168.0.0/16 by default).
 The process is automated for the vast majority of the clusters, but if you had set up custom networking settings in your cluster the upgrade might be blocked by admission controllers. If that is the case, reach out to your SA and you'll receive guidance how to move on with the upgrade. Same thing applies uf you don't want to stick with the default value and prefer to change it.
 
+#### Cilium and improved IP space usage
+
+Migration from AWS CNI to Cilium allowed us to improve the IP space delegation per WC, meaning that starting with AWS v19 release customers will be able to use full range of the CIDR instead of 25%.
+
+##### AWS-CNI in releases prior to v19
+
+By default at Giant Swarm the pod CIDR is set to 10.2.0.0/16 and can be customized in the AWSCluster CR.
+The CIDR is added in the VPC CIDR and thus cannot overlap with other CIDRs in the cluster and any other CIDR in peered VPCs.
+
+The pod CIDR space needs to be split into separate, contiguous CIDR space, one for each AZ. Since Giant Swarm supports 3 AZs, we need to divide the range into 4 subranges. In the default scenario, we end up with 4 /18 blocks (16k addresses each). One of the /18 blocks is unused meaning we have 16k (or 1/4th) IP addresses "lost".
+
+##### Cilium in releases v19 and further
+
+In v19 and further releases the default pod CIDR is 192.168.0.0/16 and is NOT part of the VPC, meaning it can overlap with mostly any other address space in the VPC (but of course not with the cluster CIDRs such as the service CIDR). The default can be changed by setting a field in the AWSCluster CR as it was the case so far.
```

Comment:
thinking about it, this is not true.
If you have pods in the same IP space as other resources in the VPC, routing will be problematic.
For example if you have a pod and a node using the same IP address, this will make networking be broken.

Reply:
so how do u want to rephrase it?

"In v19 and further releases the default pod CIDR is 192.168.0.0/16 and is NOT part of the VPC, meaning it can overlap with mostly any other address space in the VPC but not the k8s resources"?